tests/unit/ops/quantizer/test_dequantize.py

```diff
@@ -4,7 +4,7 @@
 
 import pytest
 import torch
-from deepspeed.ops import op_builder
+from deepspeed.ops.op_builder import QuantizerBuilder
```

Comment:
The chagnes in this file is actually not needed after op builder interface revert back to old style